docs/introduction/sla.md

```diff
@@ -0,0 +1,208 @@
+# Rahti Container Cloud Service Service Level Agreement DRAFT
+
+## General
+
+This Service Level Agreement (hereafter called SLA) is made between the
+customer, Rahti Container Cloud Service and the service provider, CSC - IT
+Center for Science Ltd., to cover the provision and support of the service as
+described hereafter. Amendments, comments and suggestions must be addressed
+using the communication channels defined in section 8.1. The service provider
```

Comment:
Here we can't refer to section 8.1 as vanilla markdown doesn't have numbered headings. A reference to the correct heading could be used instead.

Reply:
Fixed all the references to markdown references

Reply:
It looks like there's still two references in the text to numbered section. You can find these by searching for "section 6".

Reply:
Fixed